docs/development/codeguide.rst

```diff
@@ -168,11 +168,11 @@ Coding Style/Conventions
   addition, these checks also enforce `isort <https://pycqa.github.io/isort/>`_ to sort
   the module imports and a large set of style-checks supported by ruff_.
 
-  * We provide a ``pre-commit`` hook which automatically enforces and fixes (whenever
```

Comment:
I think this is a slight misuse of the terminology, but just to be sure I'm not the one mistaken, let's ping @nstarman